.github/workflows/build.yaml

```
@@ -88,6 +92,8 @@ jobs:
           python .pipeline/scripts/generate-release-artifacts.py --version ${{ steps.get-version.outputs.VERSION }} --path-prefix release
           zip -q -r sap-cloud-sdk-java-${{ steps.get-version.outputs.VERSION }}.zip release
           rm -rf release
+        env:
+          MVN_CLI_ARGS: ${{ env.MVN_CLI_ARGS }} -T ${{ steps.get-core-count.outputs.CORE_COUNT }}
```

```suggestion
          MVN_CLI_ARGS: ${{ env.MVN_CLI_ARGS }} -T 1C # 1 thread per cpu core
```
You can also try 2C there might be an additional performance improvement.

---

Thanks, I wasn't aware of the `1C` notation which let's maven itself figure out the core count. I would rework this PR and make it part of the global CLI args...